pkg/cvo/cvo.go

```diff
@@ -269,6 +271,31 @@ func (optr *Operator) InitializeFromPayload(restConfig *rest.Config, burstRestCo
 	return nil
 }
 
+// ownerReferenceModifier sets the owner reference to the current CV resource if no other reference exists
+func (optr *Operator) ownerReferenceModifier(object metav1.Object) {
+	var nonCVORefs []metav1.OwnerReference
+	// find any existing owner references which are not ClusterVersion resource types
```

Comment:
This is necessary I suppose because as CVO reconciles resources one which previously had CV set as the owner now has non-CV owners so we must remove CV? Can we put a comment in here to that effect or explain why this is necessary.